app/controllers/answers_controller.rb

```diff
@@ -5,6 +5,7 @@ def new
       #if all word has been answered, redirect to lesson show
       if @lesson.next_word.nil?
         @lesson.update(result: @lesson.lesson_results)
+        @lesson.lesson_activity(current_user)
```

Comment:
@ivanraybuglosa 
You can use `build_activity` method created with `has_one` association
`@lesson.build_activity(user: current_user)`